Mathlib/Algebra/Group/Hom/Defs.lean

```diff
@@ -878,6 +878,20 @@ def MulHom.inverse [Mul M] [Mul N] (f : M →ₙ* N) (g : N → M)
       _ = g (f (g x * g y)) := by rw [f.map_mul]
       _ = g x * g y := h₁ _
 
+/-- If `M` and `N` have multiplications, `f : M →ₙ* N` is a surjective multiplicative map,
+and `M` is commutative, then `N` is commutative. --/
+@[to_additive add_comm
```

Comment:
The additive name `add_comm` seems wrong.

Reply:
I'll choose `mul_comm` and `add_comm`.